tools/minikube/instant-demo.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -e
+set -u
+
+: "${USE_SSL:=false}"
+: "${backend_proto:="http"}"
```

Comment:
This shouldn't be set by a user ever. It is depending on `USE_SSL`.

Reply:
:+1: